experimental/trusted_shuffler_example/example.toml

```diff
@@ -0,0 +1,27 @@
+name = "aggregator"
+
+[backends]
+backend = { Cargo = { cargo_manifest = "examples/aggregator/backend/Cargo.toml" }, additional_args = [
+  "--grpc-listen-address=[::]:8888",
+  "--grpc-tls-private-key=./examples/certs/local/local.key",
+  "--grpc-tls-certificate=./examples/certs/local/local.pem",
+] }
+
+[applications]
+
+[applications.rust]
+type = "Classic"
+manifest = "examples/aggregator/oak_app_manifest.toml"
+out = "examples/aggregator/bin/aggregator.oak"
+
+[applications.rust.modules]
+module = { Cargo = { cargo_manifest = "examples/aggregator/module/rust/Cargo.toml" } }
```

Comment:
Why do we need an `application`? In the end we don't want one, or?

Reply:
Removed